src/Dhadgar.ShoppingCart/src/components/auth/OAuthButtonGroup.tsx

```
@@ -1,13 +1,101 @@
+              'transition-all duration-200',
+              'hover:border-text-secondary hover:bg-glow-line/20',
+              'focus:outline-none focus:ring-2 focus:ring-cyber-cyan/50 focus:ring-offset-2 focus:ring-offset-space-dark',
+              'disabled:opacity-50 disabled:cursor-not-allowed',
+              loadingProvider === provider.id && 'animate-pulse'
+            )}
+            style={{
+              '--provider-color': provider.color,
+              '--provider-glow': provider.glowColor,
+            } as React.CSSProperties}
+            title={isGaming ? provider.name : undefined}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)